uni/lib/controller/fetchers/book_fetcher.dart

```diff
@@ -0,0 +1,25 @@
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
```

Comment:
Attention! There is a view import, shouldn't be here. I think it because of the `Image` and `NetworkImage`. Try to return the all `bookInformation`, and handle the thumbnail access in the page (view)